src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
```

Comment:
Fonts should have the same `font-family` but different `font-weights` – by this you're allowed to mention `font-family` once and then change only `font-weight` if needed